index.html

```diff
@@ -418,6 +419,145 @@ <h2>Transferable MediaStreamTrack</h2>
       <p>The underlying source is supposed to be kept alive between the transfer and transfer-receiving steps, or as long as the data holder is alive.
       In a sense, between these steps, the data holder is attached to the underlying source as if it was a track.</p>
     </div>
+    <h3>MediaStreamTrack frame counters</h3>
+    <div>
+      <p>
+        Frame counters allow you to tell what the capture frame rate is, which
+        may be different from what is expected from settings - for instance, the
+        camera may be slowing down frame production because it's dark or the
+        system may be CPU starved and therefore dropping frames.
+      </p>
+    </div>
+    <div>
+      <pre class="idl"
+>partial interface MediaStreamTrack {
+  Promise&lt;MediaStreamTrackStats&gt; getStats();
+};
+</pre>
+    </div>
+    <div>
+      <p>At creation of a {{MediaStreamTrack}} object, <var>track</var>, run the
+      following steps:</p>
+      <ol>
+        <li>
+          <p>Initialize <var>track</var>.`[[FramesCaptured]]` to 0.</p>
+        </li>
+      </ol>
+    </div>
+    <div>
+      <p>Monitor frames that arrive to <var>track</var> from its source by
+      running the following in-parallel steps for every <var>frame</var>:</p>
+      <ol>
+        <li>
+          <p>If <var>frame</var> is to be discarded by <var>track</var> in
+          order to achive the frame rate decimation of <var>track</var>'s
+          settings, abort these steps.</p>
```

Comment:
Note that this wording allows the user agent to apply frame decimation earlier in the pipeline as well, since a frame not reaching the track does not trigger these in-parallel steps and that has the same behavior as aborting the steps here.